tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -153,6 +153,7 @@
             <package name="org.ballerinalang.test.variable.shadowing.*"/>
             <package name="org.ballerinalang.test.vm.*"/>
             <package name="org.ballerinalang.test.worker.*"/>
+            <package name="org.ballerinalang.test.visibility"/>
```

Comment:
Shall we add some tests for records as well. This can happen for records too right?

Reply:
Added